x-pack/plugins/fleet/public/components/agent_enrollment_flyout/agent_enrollment_flyout.test.tsx

```diff
@@ -23,11 +23,8 @@ import type { FlyOutProps } from './types';
 import { AgentEnrollmentFlyout } from '.';
 
 const render = (props?: Partial<FlyOutProps>) => {
-  cleanup();
```

Comment:
We shouldn't be cleaning up just be test starts, what's more with jest this is handled automatically for us